pkg/backend/commandapi/client.go

```diff
@@ -39,6 +40,12 @@ func (c *Command) Event(integrationName string, integrationArgs []string) protoc
 		"cmd_args_name": integrationName,
 		"cmd_args_args": fmt.Sprintf("%+v", integrationArgs),
 	}
+
+	for k, v := range metadata {
```

Comment:
do we need to have test for this change?

Reply:
https://github.com/newrelic/infrastructure-agent/pull/261/files#diff-80eaf9628da03065faf209a3465fc243089d5f3f4f87e8cdf6c191734f5aee0fR171-R175

Reply:
good one, we are not testing instrumentation events

Reply:
added into latest commit